230 INFO: PyInstaller: 6.11.1, contrib hooks: 2025.1
230 INFO: Python: 3.12.3
247 INFO: Platform: macOS-15.2-arm64-arm-64bit
247 INFO: Python environment: /Users/andrewtarasov/PycharmProjects/utilities/venv
290 WARNING: collect_dynamic_libs - skipping library collection for module '.' as it is not a package.
290 DEPRECATION: Foreign Python environment's site-packages paths added to --paths/pathex:
['venv/lib/python3.12/site-packages']
This is ALWAYS the wrong thing to do. If your environment's site-packages is not in PyInstaller's module search path then you are running PyInstaller from a different environment to the one your packages are in. Run print(sys.prefix) without PyInstaller to get the environment you should be using then install and run PyInstaller from that environment instead of this one. This warning will become an error in PyInstaller 7.0.
290 INFO: Module search paths (PYTHONPATH):
['/Library/Frameworks/Python.framework/Versions/3.12/lib/python312.zip',
 '/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12',
 '/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/lib-dynload',
 '/Users/andrewtarasov/PycharmProjects/utilities/venv/lib/python3.12/site-packages',
 '/Users/andrewtarasov/PycharmProjects/utilities/venv/lib/python3.12/site-packages/setuptools/_vendor',
 '/Users/andrewtarasov/PycharmProjects/utilities',
 '/Users/andrewtarasov/PycharmProjects/utilities/utilities',
 '/Users/andrewtarasov/PycharmProjects/utilities/venv/lib/python3.12/site-packages']
43 WARNING: discover_hook_directories: Failed to process hook entry point 'EntryPoint(name='hook-dirs', value='rapidfuzz.__pyinstaller:get_hook_dirs', group='pyinstaller40')': AttributeError: module 'rapidfuzz.__pyinstaller' has no attribute 'get_hook_dirs'
402 INFO: Appending 'datas' from .spec
407 INFO: checking Analysis
417 INFO: checking PYZ
421 INFO: EXE target arch: arm64
421 INFO: Code signing identity: None
426 INFO: checking PKG
428 INFO: Bootloader /Users/andrewtarasov/PycharmProjects/utilities/venv/lib/python3.12/site-packages/PyInstaller/bootloader/Darwin-64bit/run
428 INFO: checking EXE
